src/web/mjs/connectors/MangaDex.mjs

```diff
@@ -47,66 +47,59 @@ export default class MangaDex extends Connector {
         const regexGUID = /[a-fA-F0-9]{8}-([a-fA-F0-9]{4}-){3}[a-fA-F0-9]{12}/;
         const id = (uri.pathname.match(regexGUID) || uri.hash.match(regexGUID))[0].toLowerCase();
         const request = new Request(new URL('/manga/' + id, this.api), this.requestOptions);
-        const {data} = await this.fetchJSON(request);
+        const { data } = await this.fetchJSON(request);
```

Comment:
Use more expressive variable names, or re-use (re-assign) the same variable to prevent confusion within the same scope.
Names such as `data`, `data1`, `data2`, `data3` may be confusing for other contributors.